lxd/networks_utils.go

```diff
@@ -1017,17 +1017,55 @@ func networkUpdateStatic(s *state.State, networkName string) error {
 	return nil
 }
 
-func networkSysctl(path string, value string) error {
+func networkSysctlGet(path string) (int, error) {
+
+	// read file to memory
 	content, err := ioutil.ReadFile(fmt.Sprintf("/proc/sys/net/%s", path))
 	if err != nil {
-		return err
+		return 0, err
 	}
 
-	if strings.TrimSpace(string(content)) == value {
+	// split by newlines
+	lines := bytes.Split(content, []byte("\n"))
+
+	// get value from first line
+	value := string(bytes.TrimSpace(lines[0]))
+
+	// convert to int
+	i, err := strconv.Atoi(value)
+	if err != nil {
+		return 0, err
+	}
+
+	return i, nil
+}
+
+func networkSysctlCheckEqual(path string, value int) (bool, error) {
+
```

Comment:
We don't allow empty new lines at beginning of functions

Reply:
fixed